stubs/vobject/vobject/win32tz.pyi

```diff
@@ -0,0 +1,39 @@
+import datetime
+from typing import Any
+
+handle: Any
+tzparent: Any
+parentsize: Any
+localkey: Any
+WEEKS: Any
+
+def list_timezones(): ...
+
+class win32tz(datetime.tzinfo):
+    data: Any
+    def __init__(self, name) -> None: ...
+    def utcoffset(self, dt): ...
+    def dst(self, dt): ...
+    def tzname(self, dt): ...
+
+def pickNthWeekday(year, month, dayofweek, hour, minute, whichweek): ...
+
+class win32tz_data:
```

Comment:
Maybe everything in this file should be wrapped in `if sys.platform == "win32"`? It won't be an error if you import it in non-windows code, but it's still better than having it always accessible. Then we could also get rid of the stubtest exclude.

Reply:
Good idea. Will do when I find the time.